### DISCUSSION TITLE
[Script request] Crafty Controller

### DISCUSSION THREAD

**Discussion by CrazyWolf13** [community-scripts/ProxmoxVE#37] [Request script]
### Application Name

Crafty Controller

### Website

https://craftycontrol.com/

### Description

Crafty Controller is a free and open-source Minecraft launcher and manager that allows users to start and administer Minecraft servers from a user-friendly interface. The interface is run as a self-hosted web server that is accessible to devices on the local network by default and can be port forwarded to provide external access outside of your local network. 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.

**Comment by sTrollZ2514** [comment]
I second this one. Also (if possible) I'd like to ask for the id/pw of the user

**Comment by CrazyWolf13** [comment]
@MickLesk 
Say I'd make a PR would it be accepted, even though it's game related?

**Reply by MickLesk** [reply]
yep you can

**Comment by YamiGhor** [comment]
Please this 🙏
I would implement it on my PVE!

**Reply by CrazyWolf13** [reply]
PR is already open awaiting review.

**Reply by YamiGhor** [reply]
Sorry, but I'm a bit new on PVE, what is a PR? Do you mean I can test it to give feedback? Can you give me the link?

Edit: I see, it's a Pull Request and I have the link... Sorry

**Reply by CrazyWolf13** [reply]
https://github.com/community-scripts/ProxmoxVE/pull/1926
@YamiGhor No problem :)

PR means PullRequest, this is common with git.
That means, the script is written, working and just awaiting a last approal from other Contributors.

**Reply by YamiGhor** [reply]
Thank you @CrazyWolf13 !
I'll check the PR and see if and how I can run it.

**Reply by MickLesk** [reply]
Merged

**Comment by blablazzz** [comment]
something is not right. Script works, installs server, but then its not possible to install new server in the Crafty. Something not rights with permissions or user setup. Can anyone else check it?

**Reply by MickLesk** [reply]
Create an issue with Error description

**Reply by CrazyWolf13** [reply]
@blablazzz PR wich might fix this is already in review :)

**Reply by blablazzz** [reply]
newest script. Error while installing:
[ERROR] in line 44: exit code 0: while executing command "$@" > /dev/null 2>&1
The silent function has suppressed the error, run the script with verbose mode enabled, which will provide more detailed output.
/dev/stdin: line 17: pop_var_context: head of shell_variables not a function context
